Sources/Networking/AlamofireNetworkClient/AlamofireNetworkClient.swift

```diff
@@ -255,13 +295,13 @@ public extension AlamofireNetworkClient.Error {
   var info: ErrorInfo {
     switch self {
     case .request(_, let info),
-         .other(_, let info):
+        .other(_, let info):
       return info
     }
   }
 }
 
-// MARK: - Request API
+// MARK: - Data Request API
 public extension AlamofireNetworkClient.Request {
```

Comment:
Lets rename `Request` to `DataRequest`.

Reply:
[Done](https://github.com/poviolabs/PovioKit/pull/297/commits/4970adb066ce687b511fd049726b953438b09cb2)